sphinx/cmdline.py

```diff
@@ -83,6 +84,19 @@ def handle_exception(app, args, exception, stderr=sys.stderr):
                   file=stderr)
 
 
+def jobs_argument(value):
+    # type (int, str) -> int
+    """
+    Special type to handle 'auto' flags passed to 'sphinx-build' via -j flag. Can
+    be expanded to handle other special scaling requests, such as setting job count
+    to (cpu_count / 2)
```

Comment:
I noticed docstring says `(cpu_count / 2)`, but code uses just `cpu_count`.
I don't know which is better. so I propose you to use `cpu_count`.